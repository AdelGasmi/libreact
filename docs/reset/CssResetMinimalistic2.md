# `<CssResetMinimalistic2>`

## Usage

Simply render it in your React app

```jsx
import CssResetMinimalistic2 from 'mol-fe-react/lib/reset/CssResetMinimalistic2';

<CssResetMinimalistic2 />
```

## Contents

```js
{
  '*': {
      pad: 0,
      mar: 0,
      bd: 0,
  },
}
```
