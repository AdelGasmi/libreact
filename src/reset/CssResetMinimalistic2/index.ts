import createCssResetComponent from '../createCssResetComponent';

export const css = {
  '*': {
      pad: 0,
      mar: 0,
      bd: 0,
  },
};

const CssResetMinimalistic2 = createCssResetComponent(css);

export default CssResetMinimalistic2;
