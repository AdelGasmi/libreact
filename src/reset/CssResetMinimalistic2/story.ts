import {createElement as h} from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';
import {linkTo} from '@storybook/addon-links';
import CssResetMinimalistic2 from '.';

storiesOf('CSS resets', module)
  .add('CssResetMinimalistic2', () =>
    h(CssResetMinimalistic2)
  );
