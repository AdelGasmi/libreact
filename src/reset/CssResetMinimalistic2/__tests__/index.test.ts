import {createElement as h} from 'react';
import {renderToStaticMarkup} from 'react-dom/server';
import CssResetMinimalistic2 from '..';

describe('<CssResetMinimalistic2>', () => {
  it('renders as expected', () => {
    const html = renderToStaticMarkup(h(CssResetMinimalistic2));

    expect(html).toMatchSnapshot();
  });
});
