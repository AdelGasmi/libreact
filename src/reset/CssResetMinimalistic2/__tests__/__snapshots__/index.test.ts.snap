// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssResetMinimalistic2> renders as expected 1`] = `"<style>*{padding:0;margin:0;border:0;}</style>"`;
